draft-ietf-quic-recovery.md

```diff
@@ -1282,15 +1287,27 @@ Invoked by loss detection from DetectLostPackets when new packets
 are detected lost.
 
 ~~~
+   InPersistentCongestion(oldest_loss_time):
+     congestion_period = Now() - oldest_loss_time
```

Comment:
I think I concluded newest_lost_time - oldest_loss_time is the best option here.  Using Now() means that a single lost packet combined with some temporary ack loss causes this to trigger, which I do not believe is the intent.

Reply:
Yes, I agree. I had thought that Now() was best, but that does not account for pauses in sending... newest_lost_time - oldest_lost_time is quite good I think.